controller/src/main/java/io/pravega/controller/store/PravegaTablesScope.java

```diff
@@ -237,6 +238,33 @@ public String getName() {
                                 .thenCompose(deleted -> storeHelper.removeEntry(SCOPES_TABLE, scopeName, context.getRequestId()));
     }
 
+    @Override
+    public CompletableFuture<Void> deleteScopeRecursive(OperationContext context) {
```

Comment:
We should modify `public CompletableFuture<Void> createScope(OperationContext context) ` to ensure scope creation fails if a deletescope recursive is in progress.

Reply:
I'll be creating a new GH issue for that to indicate new status. For now I'm sending a failure in case if this method is invoked.